ivy.xml

```diff
@@ -8,7 +8,8 @@
   <dependency org="commons-codec" name="commons-codec" rev="1.7" />
   <dependency org="commons-httpclient" name="commons-httpclient" rev="3.1" />
   <dependency org="org.dom4j" name="dom4j" rev="${dom4j.version}" />
-  <dependency org="log4j" name="log4j" rev="1.2.16" />
+  <dependency org="org.apache.logging.log4j" name="log4j-core" rev="${log4j.core.version}" />
```

Comment:
I guess client side zimlet code should not depend on log4j libs, we can remove this